zerver/openapi/zulip.yaml

```diff
@@ -9523,17 +9523,17 @@ paths:
   /mobile_push/test_notification:
     post:
       operationId: test-notify
-      summary: Sends a test notification to the user's mobile device(s)
+      summary: Send a test notification to mobile device
```

Comment:
I think the "(s)" on "device(s)" is helpful, because the endpoint can be used to send to multiple devices.